biggraphite/drivers/elasticsearch.py

```diff
@@ -424,7 +424,28 @@ def get_metric(self, metric_name, touch=False):
         """See the real Accessor for a description."""
         super(_ElasticSearchAccessor, self).get_metric(metric_name, touch=touch)
         metric_name = ".".join(_components_from_name(metric_name))
-        return self._name_to_metric.get(metric_name)
+        metric = self._name_to_metric.get(metric_name)
```

Comment:
this part is copied from the "Memory" driver, it should be removed.

Reply:
Done